_posts/2020-08-18-network-upgrade-october-2020.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Monero will experience a network upgrade on 17th October 2020"
+summary: Announcement of the upcoming network upgrade (17th October)
+tags: [community]
```

Comment:
Some of the other network upgrade announcements are tagged with `[network upgrade, core, crypto, community]`

Reply:
We have a `protocol upgrade` tag, but no `network upgrade`. I will add the former and the others you suggested. Note that adding these tags is meaningless at the moment, since we don't show them anywhere.